### ISSUE TITLE
quantumarticle breaks bibtex with biber backend

### ISSUE THREAD

**Issue by cgogolin** [cgogolin/quantum-journal#22]
Unfortunately `quantumarticle` currently doesn't seem to work with the `biber` backend of `biblatex`.

The following minimal tex file:
```
\documentclass{quantumarticle}
\usepackage[backend=biber]{biblatex}
\addbibresource{quantum-bibtest.bib}
\begin{document}
\cite{Foo2035}
\printbibliography
\end{document}
```
together with this 
```
@article{Foo2035,
  author = {Foo, Bar},
  journal = {Journal},
  year = {2010},
  title = {Title},
  doi = {doi},
}
```
BibTeX file results in the following error after running `pdflatex` and then `biber`
```
Running `Biber' on `minimal' with ``biber minimal''
INFO - This is Biber 2.6
INFO - Logfile is 'minimal.blg'
ERROR - minimal.bcf is malformed, last biblatex run probably failed. Deleted minimal.bbl
INFO - ERRORS: 1

```

**Comment by cgogolin** [comment]
I have opened a question to seek advice on this on [stackexchange](http://tex.stackexchange.com/questions/343856/biblatex-produces-incomplete-bcf-with-biber-backend)

**Comment by cgogolin** [comment]
I am reopening this issue as the proposed fix creates more problems than it solves.

**Comment by cgogolin** [comment]
The hack in a8a1924534104c7d05c95af5ea394e30a2d71b4c seems to have fixed this as far as I can tell.

**Comment by cebamps** [comment]
Hello. This bug is back for me in current master.

I think the reason is that the a8a1924 hack was accidentally disabled by b405fb8. The `\ltx@ifpackageloaded` that commit introduced seems backward to me. Inverting the *if* and *else* branches makes my bcf file valid again.

https://github.com/quantum-journal/quantum-journal/blob/b6f451d670146cb188bd9909fefedeb7ca0ed8c1/quantumarticle.cls#L261-L275

**Comment by cgogolin** [comment]
Thanks for reporting and the careful analysis. I think you are right. I am sorry for introducing this stupid mistake and for not catching it with the test I am running. Please have a look at  
4526b41 and verify that this fixes the problem again.

**Comment by cebamps** [comment]
Thanks! 4526b41 is exactly what I ended up doing to fix this locally. Works great 👍

**Comment by k4rtik** [comment]
@cgogolin I see `biber` is discouraged as 
> Due to issues with `biber` we recommend to use the `bibtex` backend of `biblatex`.

Is this still an issue? If so, can the problem be documented somewhere (I cannot find what the issue is apart from this issue #22)? The `bibtex` backend is limited in several ways.

**Comment by cgogolin** [comment]
I must admit that I haven't used Biber in the last 4 years...

If proper BibLaTeX support is implemented as part of #144 and we implement some unit testing for bibtex+BibLaTeX and biber+BibLaTeX then I think we can remove this discouraging warning to the user.
